### PR TITLE
Pull Reqest for Zemismart Thread Smart Switch MT606-3

### PR DESCRIPTION
**Pull request by lelandblue**
ProductName: Zemismart Thread Smart Switch
ModelNumber: MT606-3

This fingerprint YML file was submitted in the Edge Builder Tool

### CODE REVIEW

File: drivers/SmartThings/matter-switch/fingerprints.yml [hunk 1]
```diff
@@ -71,6 +71,13 @@ matterManufacturer:
     vendorId: 0x1312
     productId: 0x01
     deviceProfileName: light-level-colorTemperature-2710k-6500k
+#Zemismart
+ - id: "5020/43524"
```
Comment (hubTab) [inline drivers/SmartThings/matter-switch/fingerprints.yml:75]:
```suggestion
  - id: "5020/43524"
```

File: drivers/SmartThings/matter-switch/fingerprints.yml [hunk 2]
```diff
@@ -71,6 +71,13 @@ matterManufacturer:
...
+    deviceLabel: "Zemismart Thread Smart Switch"
+    vendorId: 0x139C
+    productId: 0xAA04
+    deviceProfileName: "switch-3"  
+
```
Comment (hubTab) [inline drivers/SmartThings/matter-switch/fingerprints.yml:80]:
```suggestion
```

**Review by hubTab — APPROVED** [review]

**Comment by lelandblue** [comment]
@ctowns and @z-michel This is an inflight certification that is not currently Parent Child. Lets discuss next steps.

File: drivers/SmartThings/matter-switch/fingerprints.yml [hunk 3]
```diff
@@ -71,6 +71,17 @@ matterManufacturer:
     vendorId: 0x1312
     productId: 0x01
     deviceProfileName: light-level-colorTemperature-2710k-6500k
+#Zemismart
+  - id: "5020/43524"
+    deviceLabel: "Zemismart Thread Smart Switch"
+    vendorId: 0x139C
+    productId: 0xAA04
+    deviceProfileName: "switch-3"
```
Comment (lelandblue) [inline drivers/SmartThings/matter-switch/fingerprints.yml:79]: The profile assignments here need to change

**Review by ctowns — APPROVED** [review]

File: drivers/SmartThings/matter-switch/fingerprints.yml [hunk 4]
```diff
@@ -71,6 +71,17 @@ matterManufacturer:
     vendorId: 0x1312
     productId: 0x01
     deviceProfileName: light-level-colorTemperature-2710k-6500k
+#Zemismart
+  - id: "5020/43524"
+    deviceLabel: "Zemismart Thread Smart Switch"
+    vendorId: 0x139C
+    productId: 0xAA04
+    deviceProfileName: "switch-binary"
```
Comment (ctowns) [inline drivers/SmartThings/matter-switch/fingerprints.yml:79]:
```suggestion
    deviceProfileName: switch-binary 
```
nit: just formatting - all other fingerprints don't have quotes around the `deviceProfileName`

File: drivers/SmartThings/matter-switch/fingerprints.yml [hunk 5]
```diff
@@ -71,6 +71,17 @@ matterManufacturer:
...
+  - id: "5020/43524"
+    deviceLabel: "Zemismart Thread Smart Switch"
+    vendorId: 0x139C
+    productId: 0xAA04
+    deviceProfileName: "switch-binary"  
+  - id: "5020/43523"
+    deviceLabel: "Zemismart Thread Smart Switch"
+    vendorId: 0x139C
+    productId: 0xAA03
+    deviceProfileName: "switch-binary"
```
Comment (ctowns) [inline drivers/SmartThings/matter-switch/fingerprints.yml:84]:
```suggestion
    deviceProfileName: switch-binary
```
nit: same as above

**Review by ctowns — APPROVED** [review]

**Comment by lelandblue** [comment]
This PR has withdrawn its certification request... closing

**Pull request closed without merging**